js/apps/events-page/directives/event.js

```diff
@@ -0,0 +1,22 @@
+((app) => {
+	const eventDirective = (dateUtilityService, CONSTANTS) => {
+		const eventLink = ($scope) => {
+			const eventItem = $scope.eventItem;
+
+			$scope.EventScheduleString =
```

Comment:
Why uppercase for ```EventScheduleString ```?

Reply:
I got a little crazy with the "shift" key. Fixed.